tests/test_future/test_bytes.py

```diff
@@ -551,13 +554,12 @@ def test_mod(self):
         a = b % (b'seventy-nine', 79)
         self.assertEqual(a, b'seventy-nine / 100 = 79%')
 
-    @unittest.expectedFailure
     def test_imod(self):
         """
         From Py3.5 test suite (post-PEP 461)
         """
-        # if (3, 0) <= sys.version_info[:2] < (3, 5):
-        #     raise unittest.SkipTest('bytes % not yet implemented on Py3.0-3.4')
```

Comment:
I'm not sure whether these 2 tests are needed. These feature are only available after python 3.5.